cmd/node/node.go

```diff
@@ -59,6 +60,7 @@ var (
 	apiKey                                = flag.String("api-key", "", "Api key")
 	grpcAddr                              = flag.String("grpc-address", "127.0.0.1:7475", "Address for gRPC API")
 	enableGrpcApi                         = flag.Bool("enable-grpc-api", true, "Enables/disables gRPC API")
+	enableMetaMaskService                 = flag.Bool("enable-metamask", true, "Enables/disables metamask service")
```

Comment:
I think `metaMaskServiceAddr` parameter would be useful (Address for metamask service).